kuma/landing/views.py

```diff
@@ -10,6 +11,7 @@
 from kuma.search.serializers import GroupWithFiltersSerializer
 
 
+@cache_page(60 * 15)
```

Comment:
Setting a short timeout like this should work fine with the caching that happns on the load balancer every 60 minutes. So we can say that changes that impact things on the frontpage will take a minimum of 15 minutes and a maximum of 60 minutes to reach it.

What I want to add though is an optional parameter to the decorator called `key_prefix` that would allow us to invalidate the frontpage cache on deploy if we ever need an urgent change to apply for certain. That's really needed since by default Django's cache middleware (the backend of the decorator) doesn't take a build ID or something into account. So let's add this:

    @cache_page(60 * 15, key_prefix='v1')
    def home(request):
        # ...

Or even better move both values into settings so we have all cache related values in one spot:

    @cache_page(HOMEPAGE_CACHE_TIMEOUT, key_prefix=HOMEPAGE_CACHE_PREFIX)
    def home(request):
        # ...

Then in `settings.py` next to the `CACHES` setting:

    HOMEPAGE_CACHE_TIMEOUT = 60 * 15
    HOMEPAGE_CACHE_PREFIX = 'v1'